server/src/main/java/org/jboss/as/server/operations/AbstractInstallationReporter.java

```diff
@@ -310,11 +316,23 @@ String getInstallationDir() {
         }
 
         String getLastUpdateDate() {
-            if (patchingInfo.isDefined()) {
-                List<ModelNode> result = Operations.readResult(patchingInfo).asList();
-                for (ModelNode patchAtt : result) {
-                    if (patchAtt.has("applied-at")) {
-                        return patchAtt.get("applied-at").asString();
+            if (installerInfo.get("result").isDefined()) {
+                List<ModelNode> result = Operations.readResult(installerInfo).asList();
+                for (ModelNode installerAtt : result) {
+                    if (installerAtt.has("timestamp")) {
+                        return installerAtt.get("timestamp").asString();
```

Comment:
We should keep the format given by the legacy tool:

https://github.com/wildfly/wildfly-core/blob/69e4d2a6856d4dd909ac02400a77a1576ef4e667/patching/src/main/java/org/jboss/as/patching/runner/PatchUtils.java#L200

Reply:
OK that probably should happen in [InstMgrHistoryHandler](https://github.com/wildfly/wildfly-core/blob/main/installation-manager/src/main/java/org/wildfly/core/instmgr/InstMgrHistoryHandler.java#L54), in the Reporter it's too late - we already have a text value